src/commands/Moderation/dehoist.ts

```diff
@@ -55,16 +69,25 @@ export default class extends SkyraCommand {
 		});
 	}
 
+	private shouldDehoist([memberId, memberTag]: [string, MemberTag]) {
+		const displayName = memberTag.nickname ?? this.client.userTags.get(memberId)!.username;
+		if (!displayName) return false;
+
+		const char = displayName.codePointAt(0)!;
+
+		// number char codes are greater than kLowestCode, but still hoist the
+		if (char >= this.kLowestCode && !(char >= kLowestNumberCode && char <= kHighestNumberCode)) return false;
+		return true;
```

Comment:
You can negate the if and return it instead of adding extra conditionals.

Reply:
I thought about doing that, but this way is a lot more readable imo.